src/plots/layout_attributes.js

```diff
@@ -183,6 +183,7 @@ module.exports = {
         'annotations': 'annotations',
         'shapes': 'shapes',
         'images': 'images',
-        'updatemenus': 'updatemenus'
+        'updatemenus': 'updatemenus',
+        'slider': 'slider'
```

Comment:
@rreusser this line here should be `sliders: sliders`.

Why you ask? It is to make `Plotly.PlotSchema.get()` include `layout.sliders` attributes. The `PlotSchema` module looks for the component by name [here](https://github.com/plotly/plotly.js/blob/fce36fa46bd53fa5ebe4e4892479258040783409/src/plot_api/plot_schema.js#L242) to paste nested attributes in the plot schema object.